Makefile

```diff
@@ -373,8 +374,37 @@ test:
 	fi; \
 	PYTHONDONTWRITEBYTECODE=1 py.test -p no:cacheprovider -n auto $(TEST_DIRS)
 	cd awxkit && $(VENV_BASE)/awx/bin/tox -re py2,py3
+	make test_collection_all
```

Comment:
we will also remove this line once the test runs and confirms the new task is working.